content/docs/lists-and-keys.md

```diff
@@ -296,6 +297,7 @@ function NumberList(props) {
 }
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/BLvYrB?editors=0010)
+[**Փորձել CodePen-ում**](https://codepen.io/gaearon/pen/BLvYrB?editors=0010)
 
-Sometimes this results in clearer code, but this style can also be abused. Like in JavaScript, it is up to you to decide whether it is worth extracting a variable for readability. Keep in mind that if the `map()` body is too nested, it might be a good time to [extract a component](/docs/components-and-props.html#extracting-components).
+
+Երբեմն, որպես արդյունք, ունենում ենք ավելի մաքուր կոդ, սակայն այս ոճը կարող է չարաշահվել։ Ինչպես JavaScript-ում, ձեր հաեցողության տակ է, որոշել տարանջատել փոփոխականը ավելի ընթերնելի դարձնելու համար, թե ոչ։ Մտապահեք. եթե map()-ի մարմինը բավականին ներդրված է, ապա միգուցե կարիք կա [կոմպոնենտը տարանջատելու](/docs/components-and-props.html#extracting-components)։
```

Comment:
մի տող ավել է ավելացված․․․ 
Երբեմն սա բերում է ավելի մաքուր կոդի, սակայն այս ոճը նույնպես կարող է չարաշահվել։ Ինչպես JavaScript-ում, այստեղ նույնպես ձեր հայեցողության տակ է որոշելը՝ տարանջատել փոփոխականը՝ ավելի ընթեռնելի դարձնելու համար, թե՝ ոչ։ Մտապահեք, որ եթե map()-ի մարմինը բավականին ներդրված է, ապա գուցե կարիք կա [կոմպոնենտը տարանջատելու](/docs/components-and-props.html#extracting-components)։